docs/intro/install.rst

```diff
@@ -200,16 +200,16 @@ solutions:
 
       brew install python
 
-  * Latest versions of python have ``pip`` bundled with them so you won't need
+  * Later versions of python have ``pip`` bundled with them, so you won't need
     to install it separately. If this is not the case, upgrade python::
 
       brew update; brew upgrade python
```

Comment:
Maybe we should remove this bullet point altogether. I imagine all supported versions of Python (3.8+) have pip bundled.

Reply:
Yeah I just checked, you're right